examples/player/simid_player.html

```diff
@@ -100,10 +118,13 @@
     </table>
     <button onclick="playAd()"> Start ad playback </button>
     <button onclick="initAd()"> Initialize the creative </button>
+    <button onclick="closeAd()"> Close the ad </button> 
+    <button onclick="skipAd()"> Skip ad </button>
+    <button onclick="fatalError()"> Fatal Error </button>
   </div>
 
   <div id="player_div" style="position:absolute;">
-    <video id="video_player" class="content" src="../media/Big_Buck_Bunny.mp4" controls> </video>
+    <video id="video_player" class="content" src="../media/fakeContent.mp4" controls> </video>
```

Comment:
Why switch? Is it just a smaller file?
Maybe we should get rid of the old media file then while were here?

Reply:
smaller file - the bunny one was making the player act weirdly

Reply:
Yeah, there was an issue with local python players not being able to handle the large video file.